shaders/mobile_nerf_rayquery/rayquery_morpheus_combo.frag

```diff
@@ -0,0 +1,343 @@
+/* Copyright (c) 2024, Qualcomm Innovation Center, Inc. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 the "License";
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * ------------------------------------------------------------------------
+ *
+ * THIS IS A MODIFIED VERSION OF THE ORIGINAL FILE
+ *
+ * The original file, along with the original Apache-2.0 LICENSE can be found at:
+ * https://github.com/google-research/jax3d/tree/main/jax3d/projects/mobilenerf
+ *
+ * Modification details: Shader code was updated to work on Vulkan (originally
+ * built for WebGL)
+ * Contributor: (Qualcomm) Rodrigo Holztrattner - quic_rholztra@quicinc.com
+ */
+#version 460
+#extension GL_EXT_scalar_block_layout : enable
+#extension GL_EXT_ray_query : enable
+#extension GL_EXT_nonuniform_qualifier : enable
+
+// Uncomment this to use opaque mode
+// Should be faster, but would get VK_ERROR_DEVICE_LOST on some AMD devices
```

Comment:
you mention the device_lost issue here, but below you mention it's due to compatibility with the original models?

Reply:
Answered below, happy to address in a future PR